src/styles/atoms/_card.scss

```
@@ -0,0 +1,3 @@
+.c-card {
+  background-color: white;
```

Just so we are consistent with using the color function for our color mappings 👍 

```suggestion
  background-color: color(white);
```

---

sure… did not see white defined in the colours.scss 👍